lib/bike_brigade_web/live/live_helpers.ex

```diff
@@ -31,6 +31,7 @@ defmodule BikeBrigadeWeb.LiveHelpers do
     |> Calendar.strftime("%x %-I:%M%p %Z")
   end
 
+  # TODO: move this!
```

Comment:
Is this a `TODO` for another time or for this PR?

Reply:
another time :)